index.html

```diff
@@ -14,61 +14,80 @@
   <![endif]-->
 </head>
```

Comment:
Don't forget to add the meta tag with viewport attribute settings so that media queries will work properly with other devices.

```
<meta name="viewport" content="width=device-width, initial-scale=1">
```